app/routes.py

```diff
@@ -27,22 +30,61 @@ def userRoute(id):
     elif request.method == 'DELETE':
         return user_controller.delete_user(id)
 
+
 @app.route('/login', methods=['POST'])
 def logins():
     return user_controller.login()
 
+
 @app.route('/category', methods=['GET', 'POST'])
 @jwt_required()
 def category():
     if request.method == 'GET':
         return category_controller.get_all_category()
     elif request.method == 'POST':
         return category_controller.create_category()
-    
+
+
 @app.route('/category/<id>', methods=['GET', 'DELETE'])
 @jwt_required()
 def categoryRoute(id):
     if request.method == 'GET':
         return category_controller.get_category_by_id(id)
     elif request.method == 'DELETE':
-        return category_controller.delete_category(id)
\ No newline at end of file
+        return category_controller.delete_category(id)
+
+
+# create score and get all scores routes
+@app.route('/score', methods=['GET', 'POST'])
+@jwt_required()
+def score():
+    if request.method == 'GET':
+        return score_controller.get_all_scores()
+    elif request.method == 'POST':
+        return score_controller.create_score()
+
+
+# get scores by user id
+@app.route('/scores_user_id', methods=['GET'])
```

Comment:
when it's a GET method, use param to place the id, not in the form or body

Reply:
I have fixed it